### PR TITLE
Added the missed route-map calls for idf isolation

### PR DESCRIPTION
**Pull request by tjchadaga**
#### Why I did it
Fixes https://github.com/sonic-net/sonic-buildimage/issues/19380
To enable idf_isolation route-map towards BGP peer

##### Work item tracking
- Microsoft ADO **(number only)**: 28896562

#### How I did it
Added the missing calls to route-map and updated UT

#### How to verify it
Verify that https://github.com/sonic-net/sonic-buildimage/issues/19380 is fixed

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111
- [ ] 202205
- [ ] 202211
- [ ] 202305

#### Tested branch (Please provide the tested image version)

- [ ] 
- [ ]

### CODE REVIEW

**Comment by abdosi** [comment]
is this master/202405  only issue ?

File: dockers/docker-fpm-frr/frr/bgpd/templates/general/policies.conf.j2 [hunk 1]
```diff
@@ -59,6 +60,9 @@ route-map FROM_BGP_PEER_V6 permit 1
 route-map FROM_BGP_PEER_V6 permit 100
 !
 route-map TO_BGP_PEER_V6 permit 100
+  call CHECK_IDF_ISOLATION
+!
+route-map CHECK_IDF_ISOLATION permit 10
```
Comment (judyjoseph) [inline dockers/docker-fpm-frr/frr/bgpd/templates/general/policies.conf.j2:65]: Why we to define CHECK_IDF _ISOLATION with permit 10 here, see it is present in dockers/docker-fpm-frr/frr/bgpd/idf_isolate/idf_isolate.conf.j2

Comment (tjchadaga) [reply dockers/docker-fpm-frr/frr/bgpd/templates/general/policies.conf.j2:65]: this route-map with permit 10 is the default configuration that gets invokes by the "call". idf_isolate.conf.j2 gets rendered only when idf_isolation is executed.

**Comment by tjchadaga** [comment]
> is this master/202405 only issue ?

yes

**Review by judyjoseph — APPROVED** [review]

**Review by arlakshm — APPROVED** [review]

**Comment by arlakshm** [comment]
@lguohan, can you help merge this PR

**Comment by bingwang-ms** [comment]
@StormLiangMS FYI

**Pull request merged**